src/Extensions/DataListExtension.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace SilverStripe\Versioned\Extensions;
+
+use RuntimeException;
+use SilverStripe\Core\Extension;
+use SilverStripe\ORM\DataObject;
+use SilverStripe\Versioned\Versioned;
+
+class DataListExtension extends Extension
```

Comment:
```suggestion
/**
 * @extends Extension<DataList>
 */
class DataListExtension extends Extension
```
And add the associated `use` statement for `DataList` so IDEs know what class that is.

Reply:
Updated